cmd/umserver/graceful_shutdown.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"io"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+)
+
+// closingStructure includes all components, which are connected with our application
+type closingStructure struct {
```

Comment:
Naming like `closingStructure`, `closingMap` or `closingSlice` is anti-pattern

Reply:
You are right) Fixed it